src/Engine/Objects/Actor.cpp

```diff
@@ -3006,6 +3019,7 @@ bool Actor::isActorKilled(ACTOR_KILL_CHECK_POLICY policy, int param, int count)
 
 int Actor::searchDeadActorsByID(int *pTotalActors, int id) {
     *pTotalActors = 0;
+    // TODO(pskelton): double '!' ?
```

Comment:
Might have been added by me, `!!` is essentially a shorthand for `static_cast<bool>`

Reply:
Ah makes sense